src/BlazorWebView/src/core/build/Microsoft.AspNetCore.Components.WebView.Maui.props

```diff
@@ -0,0 +1,2 @@
+﻿<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+</Project>
```

Comment:
Do we need this?

Reply:
At one point it was needed, but I kept it there for the sake of symmetry. I'll delete in a follow-up if we end up not using it.